ambry-store/src/main/java/com.github.ambry.store/DiskSpaceAllocator.java

```diff
@@ -63,8 +63,12 @@
   private final File swapReserveDir;
   private final long requiredSwapSegmentsPerSize;
   private final StorageManagerMetrics metrics;
+  // Each store's reserve pool should be thread-safe as well because different threads may perform operation on same store.
+  // For example, request handler thread and compaction manager thread operates on same store.
   private final Map<String, ReserveFileMap> storeReserveFiles = new HashMap<>();
```

Comment:
What's the relevancy of this comment to the variables below?  It would make sense if you were changing them to ConcurrentHashMap but I don't understand the comment without any related changes

Reply:
Sorry for misleading comment here. I was trying to say reserve pool of each store should be thread-safe and that's the reason why I used `ReserveFileMap` instead of a simple hashmap here because `ReserveFileMap` is a thread-safe data structure.
I will update the comment.